ERRORCODES.md

```diff
@@ -79,3 +80,4 @@ recommend only using `--extend-select`, never `--select`.
 | Code | Description
 |------|------------
 | Y090 | `tuple[int]` means "a tuple of length 1, in which the sole element is of type `int`". Consider using `tuple[int, ...]` instead, which means "a tuple of arbitrary (possibly 0) length, in which all elements are of type `int`".
+>>>>>>> upstream/main
```

Comment:
```suggestion
```

Reply:
I swear I looked at the diff before making the PR :)